docs/resources/chain-integration.md

```diff
@@ -46,6 +46,268 @@
+// Request 
+curl -X POST --data '{"jsonrpc":"2.0","method":"net_listening","params":[],"id":1}' -H "Content-Type: application/json" http://localhost:8545
+// Result
+{"jsonrpc":"2.0","id":1,"result":true}
+```
+
+### Eth Methods
+
+### `eth_protocolVersion`
+
+Returns the current ethereum protocol version.
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_protocolVersion","params":[],"id":1}' -H "Content-Type: application/json" http://localhost:8545
+// Result
+{"jsonrpc":"2.0","id":1,"result":"0x3f"}
+```
+
+### `eth_syncing`
+
+The sync status object may need to be different depending on the details of Tendermint's sync protocol. However, the 'synced' result is simply a boolean, and can easily be derived from Tendermint's internal sync state.
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_syncing","params":[],"id":1}' -H "Content-Type: application/json" http://localhost:8545
+// Result
+{"jsonrpc":"2.0","id":1,"result":false}
+```
+
+### `eth_gasPrice`
+
+Returns the current gas price in the default EVM denomination parameter.
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_gasPrice","params":[],"id":1}' -H "Content-Type: application/json" http://localhost:8545
+// Result
+{"jsonrpc":"2.0","id":1,"result":"0x0"}
+```
+
+### `eth_accounts`
+
+Returns array of all accounts owned by the client.
+
+```json
+// Request
+curl -X POST --data '{"jsonrpc":"2.0","method":"eth_accounts","params":[],"id":1}' -H "Content-Type: application/json" http://localhost:8545
+// Result
+{"jsonrpc":"2.0","id":1,"result":["0x3b7252d007059ffc82d16d022da3cbf9992d2f70","0xddd64b4712f7c8f1ace3c145c950339eddaf221d","0x0f54f47bf9b8e317b214ccd6a7c3e38b893cd7f0"]}
+```
+
+## JSON-RPC Server
+### Pre-requisite Readings
+
+- [EthWiki JSON-RPC API](https://eth.wiki/json-rpc/API) {prereq}
+- [Geth JSON-RPC Server](https://geth.ethereum.org/docs/rpc/server) {prereq}
+
+### JSON-RPC API
+
+[JSON](https://json.org/) is a lightweight data-interchange format. It can represent numbers, strings, ordered sequences of values, and collections of name/value pairs.
+
+[JSON-RPC](http://www.jsonrpc.org/specification) is a stateless, light-weight remote procedure call (RPC) protocol. Primarily this specification defines several data structures and the rules around their processing. It is transport agnostic in that the concepts can be used within the same process, over sockets, over HTTP, or in many various message passing environments. It uses JSON ([RFC 4627](https://www.ietf.org/rfc/rfc4627.txt)) as data format.
+
+### JSON-RPC Support
+
+Ethermint supports all standard web3 JSON-RPC APIs. You can find documentation for these APIs on the [`JSON-RPC Methods`](./endpoints.md) page.
+
+JSON-RPC is provided on multiple transports. Ethermint supports JSON-RPC over HTTP and WebSocket. Transports must be enabled through command-line flags or through the `app.toml` configuration file. For more details see the []
```

Comment:
Expecting a link to some details pages?

Reply:
added link and changed wording to section since the endpoints are on the same page but different section